autodispose-rxlifecycle-interop/src/main/java/com/ubercab/autodispose/rxlifecycleinterop/RXLifeCycleInterop.java

```diff
@@ -0,0 +1,54 @@
+package com.ubercab.autodispose.rxlifecycleinterop;
+
+import com.trello.rxlifecycle2.LifecycleProvider;
+import com.trello.rxlifecycle2.OutsideLifecycleException;
+import com.trello.rxlifecycle2.RxLifecycle;
+import com.uber.autodispose.AutoDispose;
+import com.uber.autodispose.LifecycleEndedException;
+import com.uber.autodispose.ScopeProvider;
+import io.reactivex.Maybe;
+
+/**
+ * Interop class for {@link RxLifecycle}. It provides static utility methods to convert {@link
```

Comment:
Nit: don't link RxLifecycle since you're talking about the library and not the class. Same with the other link below